### PR TITLE
Some bugs related to workgroup on latest test

### PR DESCRIPTION
**Pull request by hoangdat**

### CODE REVIEW

File: lib/presentation/util/widget_utils.dart [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * applicable Additional Terms for LinShare along with this program. If not, see
+ * <http://www.gnu.org/licenses/> for the GNU Affero General Public License version
+ *  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
+ *  the Additional Terms applicable to LinShare software.
+ */
+
+import 'package:flutter/widgets.dart';
+
+class WidgetConstant {
+  static final Widget NO_WIDGET = null;
```
Comment (dab246) [inline lib/presentation/util/widget_utils.dart:37]: I don't think it is necessary to assign null

**Review by dab246 — COMMENTED** [review]
LGTM

File: lib/presentation/util/widget_utils.dart (hunk 1 above)
Comment (hoangdat) [reply lib/presentation/util/widget_utils.dart:37]: It is readable :)

**Review by huynguyennovem — COMMENTED** [review]
/lgtm

**Pull request merged**